messenger/make.py

```diff
@@ -67,9 +76,55 @@ def build_octave():
     do_build(make_cmd, 'messenger.mex')
 
 
+def which_matlab():
```

Comment:
Does pexpext work if it's not in the path?

In my which function I look in the default installation location for MATLAB
even if it's not in the path. Annoying because MATLAB uses a different
structure on every platform. It would be great if pexpect could replace
this.
On Apr 10, 2015 4:09 PM, "Ariel Rokem" <notifications@github.com> wrote:

> In messenger/make.py
> <https://github.com/arokem/python-matlab-bridge/pull/178#discussion_r28187251>
> :
>
> > @@ -67,9 +76,55 @@ def build_octave():
> >      do_build(make_cmd, 'messenger.mex')
> >
> >
> > +def which_matlab():
>
> This will all be replaced by the pexpect stuff that @blink1073
> <https://github.com/blink1073>. Does that work well on Windows?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/arokem/python-matlab-bridge/pull/178/files#r28187251>.
>